test/lsp/omni.vimspec

```diff
@@ -185,19 +185,19 @@ Describe lsp#omni
 
         It should sort by sortText
             let items = [{
-            \ 'label': 'my-label3',
+            \ 'label': 'my-label1',
             \ 'kind': '3',
-            \ 'sortText': '3'
+            \ 'sortText': 'c'
             \},
             \{
-            \ 'label': 'my-label1',
+            \ 'label': 'my-label2',
             \ 'kind': '3',
-            \ 'sortText': '1'
+            \ 'sortText': 'a'
             \},
             \{
-            \ 'label': 'my-label2',
+            \ 'label': 'my-label3',
             \ 'kind': '3',
-            \ 'sortText': '2'
+            \ 'sortText': 'b'
```

Comment:
Previously these tests passed because number strings like `'1'` are implicitly converted to numbers like `1`.